_layouts/tutorial_hands_on.html

```diff
@@ -59,10 +59,11 @@
                 <ul class="navbar-nav">
                     <li class="nav-item">
                         <a class="nav-link" href="{{ site.baseurl }}/topics/{{ topic.name }}" title="Go back to list of tutorials">
-                            {% icon topic %} {{ topic.title }}
+                            {% icon topic %} Back to Topic
                         </a>
                     </li>
 
+                    <!-- language selector moved to help menu
```

Comment:
this got moved and is working now elsewhere -- maybe just delete the section rather than commenting it?